src/key_file.rs

```diff
@@ -0,0 +1,61 @@
+// Copyright 2015-2016, The Gtk-rs Project Developers.
+// See the COPYRIGHT file at the top-level directory of this distribution.
+// Licensed under the MIT license, see the LICENSE file or <http://opensource.org/licenses/MIT>
+
+use libc;
+use ffi;
+use translate::*;
+use std;
+use std::ptr;
+use std::path;
+use error::Error;
+use auto::KeyFileFlags;
+
+use KeyFile;
+
+impl KeyFile {
+
+    pub fn error_quark() -> ffi::GQuark {
+        unsafe {
+            ffi::g_key_file_error_quark()
+        }
+    }
```

Comment:
Oh, I guess we've overlooked this bit. The function doesn't need to be exposed, the generated enum whenever it gets added covers that.